### PR TITLE
Work with raw data

### PR DESCRIPTION
**Pull request by HoretskaKris**
1. Conducted initial data analysis
2. Created an algorithm for data cleaning and enrichment
3. Data cleaning and enrichment work

### CODE REVIEW

File: .ipynb_checkpoints/README-checkpoint.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Apartment_Prices_in_KR-PL-
+Apartment Prices in Poland KRA
```
Comment (uItramarine) [inline .ipynb_checkpoints/README-checkpoint.md]: @HoretskaKris  Я так розумію зміни до цього файлу будуть пізніше?

File: .ipynb_checkpoints/Untitled-checkpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+ "cells": [],
+ "metadata": {},
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```
Comment (uItramarine) [inline .ipynb_checkpoints/Untitled-checkpoint.ipynb]: File name?

File: normalize_and_clean_data.py [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    df = fill_condition(df)
+    df = fill_missing_building_material(df)
+    df = fill_missing_type(df)
+    df = fill_missing_build_year(df)
+    df = fill_missing_floor_count(df)
+    df = fill_missing_floor(df)
+    df = fill_missing_distances(df)
+    df = fill_missing_has_elevator(df)
+    df = replace_yes_no(df)
+    df = replace_ownership_value(df)
```
Comment (uItramarine) [inline normalize_and_clean_data.py]: Check if this methods inplace. https://www.geeksforgeeks.org/inplace-vs-standard-operators-python/.

I am not shure but it seams like you overrite df which changed values to same values. Check it!

**Review by uItramarine — CHANGES_REQUESTED** [review]
1. Connect service https://www.reviewnb.com/ to repo. It will help me to review changes in ipynb files. (_For eamxple file **data_quality_analysis.ipynb**_)
2. Why files from /_raw_data_ in branch? Ignore them and add info about source off code in readme file.
3. You can write comments on english from start. Waste of time change them after. Or delete them at all. 
4. Nice size of PR(pull request). (_under 300 lines of code_). There is statistic that show quaity of review comparing to lines of code [here](https://www.usebubbles.com/blog/code-reviews?utm_medium=organic&utm_source=www.google.com&utm_content=blog%252Fcode-reviews).

File: .ipynb_checkpoints/README-checkpoint.md (hunk 1 above)
Comment (HoretskaKris) [reply .ipynb_checkpoints/README-checkpoint.md]: yes

File: .ipynb_checkpoints/Untitled-checkpoint.ipynb (hunk 2 above)
Comment (HoretskaKris) [reply .ipynb_checkpoints/Untitled-checkpoint.ipynb]: del this file

File: normalize_and_clean_data.py (hunk 3 above)
Comment (HoretskaKris) [reply normalize_and_clean_data.py]: +

File: data_quality_analysis.ipynb [hunk 4]
```diff
@@ -0,0 +1,533 @@
+{
```
Comment (uItramarine) [inline data_quality_analysis.ipynb:1]: <p>One of the best anylysis that i have ever seen. :))</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HoretskaKris/Apartment_Prices_in_KR-PL-/pull/1/discussion/#comment-1825862321'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='23'/>

**Review by uItramarine — APPROVED** [review]
LGTM!

**Pull request merged**